app/models/ont/run_factory.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+# Ont namespace
+module Ont
+  # RunFactory
+  # The factory will create runs from a list of libraries
+  class RunFactory
+    include ActiveModel::Model
+
+    validate :check_run, :check_flowcells
+
+    def initialize(library_names = [])
+      @flowcells = []
+      build_run(library_names)
```

Comment:
See position comment from #360

Reply:
This was because the destination branch hadn't been merged back into this one yet.  I've done this and that should resolve your concerns.